src/lib/phdi-building-blocks/phdi_building_blocks/fhir.py

```diff
@@ -125,3 +128,184 @@ def upload_bundle_to_fhir_server(bundle: dict, access_token: str, fhir_url: str)
     except Exception:
         logging.exception("Request to post Bundle failed for json: " + str(bundle))
         return
+
+
+def export_from_fhir_server(
+    access_token: str,
+    fhir_url: str,
```

Comment:
The `fhir.py` file is getting pretty long. Wondering if we should move all of the export functions and functionality into their own file. Thoughts?